Makefile

```diff
@@ -0,0 +1,24 @@
+# Project packages.
+PACKAGES=$(shell go list ./... | grep -v /vendor/)
+
+# Flags passed to `go test`
+TESTFLAGS ?= 
+
+.PHONY: all build test coverage
+.DEFAULT: all
+
+all: build
+
+build: ## no binaries to build, so just check compilation suceeds
+	go build ${PACKAGES}
```

Comment:
In which case, we should probably keep it simple, and remove the `PACKAGES` altogether.

Reply:
Like this?